aries-site/src/examples/cardPreviews/page-layouts/StickyHeaderPreview.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Box } from 'grommet';
+
+export const StickyHeaderPreview = () => {
+  return (
+    <Box fill direction="row" gap="xsmall">
+      <Box
+        basis="20px"
```

Comment:
Would we be able to get away with using `theme.global.edgeSize.medium` instead of `20px` here? That would resolve to 24px and keep all spacing in sync.

If not, consider enumerating the 20px in a variable since it is being used multiple times for the same type of usage.

Reply:
Good call, let me try out the edgeSize first and if not I agree variable is good second option.